etc/tools/runsplitter/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+SET(name "EURunsplitter.exe")
+
```

Comment:
We did not change it yet for the 1.X branch but the explicite file ending should be removed.